Delta/Settings/SettingsViewController.swift

```diff
@@ -99,6 +99,12 @@ class SettingsViewController: UITableViewController
             self.versionLabel.text = NSLocalizedString("Delta", comment: "")
             #endif
         }
+        
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(SettingsViewController.handleTapGesture(_:)))
+        tapGesture.numberOfTapsRequired = 13
```

Comment:
Happy to have this feature hidden for now, but 13 might be a bit high even for a hidden feature. IMO ~3 taps would be enough for a relatively harmless feature like this.
```suggestion
        tapGesture.numberOfTapsRequired = 3
```